src/python/grpcio/grpc/_channel.py

```diff
@@ -99,6 +99,22 @@ def _wait_once_until(condition, until):
     else:
       condition.wait(timeout=remaining)
 
+def _check_call_error(call_error, metadata):
+  if call_error == cygrpc.CallError.ok:
+    return
+  elif call_error == cygrpc.CallError.invalid_metadata:
+    raise ValueError("metadata was invalid: %s" % metadata)
```

Comment:
Maintain consistent use of single-quote as the string delimiter throughout this file.